### ISSUE TITLE
Framework dSYM

### ISSUE THREAD

**Issue by drewcrawford** [agentsim/Symbolicator#2]
Hello,

I've got a crash inside a framework that symbolicator doesn't handle correctly.  I was able to manually symbolicate it with `atos`.

If you're still maintaining this tool, I'd be happy to provide an affected crashlog and enough information to reproduce.

**Comment by agentsim** [comment]
Yes, if you could attach it to the bug report that would be great.

**Comment by drewcrawford** [comment]
I can't attach arbitrary files to github issues, but I sent via email.

Some notes about this setup:

This is a crash from inside a unit test of a Mac framework.  The "executable" in this case is `xctest`, which loads the actual test target itself, in this case `NitrogenBench.xctest`.  That (test target) is where the crash actually occurred.

I can symbolicate this file by hand by using

``` bash
xcrun atos -o /path/to/NitrogenBench.xctest/Contents/MacOS/NitrogenBench -l 0x10cda6000 0x000000010cdab74f
@objc NitrogenBench.TCPBenches.testBenchmarkLatencyMS (NitrogenBench.TCPBenches)() -> () (in NitrogenBench) (TCPBenches.swift:26)
```

Which as you can see produces a reasonable source code listing for the top item in the stack trace.

**Comment by agentsim** [comment]
Looks like the dwarf line number symbols are stored differently. Maybe because it is a Swift compilation. I'll have to dig a bit further to fix this one.

**Comment by drewcrawford** [comment]
May also be tooling-related.  This sample is from

Xcode Version 7.0 beta (7A120f)
OSX 10.11 (15A178w)

**Comment by chsculpt** [comment]
Does having Beta version of Xcode 7 installed along side 6.4 affect using this tool?   It seems like my files aren't symbolicating successfully in the last few weeks.

**Comment by agentsim** [comment]
It is quite possible. Now that the GM is released, I'll try updating the dwarf library and see if that helps with resolving the symbols.

**Comment by chsculpt** [comment]
Are you familiar with this?
https://github.com/MaheshRS/symbolication-plugin

I found this will searching for ways to symbolicate.   It worked pretty well but sort of makes me nervous adding a third party plugin to Xcode.

**Comment by agentsim** [comment]
Haven't heard of that, but if it works I wouldn't be too worried about it being a plugin. I use a few plug-ins everyday with Xcode (I'd be lost without XVim!)
